tiatoolbox/dataloader/wsireader.py

```diff
@@ -1054,3 +1016,96 @@ def info(self):
         )
 
         return param
+
+
+class VFReader(WSIReader):
+    """Class for reading small visual field images.
+
+    Supported formats:
+
+    - .jpg
+    - .png
+
+    Attributes:
+        img (ndarray)
+
+    """
+
+    def __init__(self, input_path="."):
+        super().__init__(input_path=input_path,)
+
+        self.img = misc.imread(self.input_path)
+
+    @property
+    def info(self):
+        """Visual Field meta data reader.
+
+        Returns:
+            WSIMetadata: containing meta information.
+
+        """
+        param = WSIMeta(
+            file_path=self.input_path,
+            objective_power=None,
+            slide_dimensions=self.img.shape[:-1],
+            level_count=1,
+            level_dimensions=(self.img.shape[:-1],),
+            level_downsamples=[1.0],
+            vendor=None,
+            mpp=None,
+            raw=None,
+        )
+
+        warnings.warn("Unknown scale (no objective_power or mpp).")
+
+        warnings.warn("Raw data is None.")
+
+        return param
+
+    def read_rect(self, location, size, resolution=0, units="level"):
+        # Find parameters for optimal read
+        (
+            _,
+            level_location,
+            _,
+            post_read_scale,
+            baseline_read_size,
+        ) = self._find_read_rect_params(
+            location=location, size=size, resolution=resolution, units=units,
+        )
+
+        im_region = self.img[
+            level_location[1]:level_location[1] + baseline_read_size[1],
+            level_location[0]:level_location[0] + baseline_read_size[0],
+            :,
+        ]
+
+        if np.any(post_read_scale != 1.0):
+            interpolation = cv2.INTER_AREA
+            if np.any(post_read_scale > 1.0):
+                interpolation = cv2.INTER_CUBIC
+            im_region = cv2.resize(im_region, size, interpolation=interpolation)
+
+        im_region = transforms.background_composite(image=im_region)
+        return im_region
+
+    def read_bounds(self, bounds, resolution=0, units="level"):
```

Comment:
I have discussed this with John we can use `mpp` and `objective` in future by linking metadata for tiles as well. So I will keep it for now. For now it will generate error as the meta for `mpp` and `objective` is not there. It behaves in a similar way to JP2 and OpenSlide Readers so the documentation in base class is enough I think.